x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -15,6 +15,9 @@ export const telemetryIndexPattern = 'metrics-endpoint.telemetry-*';
 export const LIMITED_CONCURRENCY_ENDPOINT_ROUTE_TAG = 'endpoint:limited-concurrency';
 export const LIMITED_CONCURRENCY_ENDPOINT_COUNT = 100;
 
+export const HOST_METADATA_LIST_API = '/api/endpoint/metadata';
```

Comment:
Now that we have these defined here, we should also refactor the server code to use them - here:

https://github.com/elastic/kibana/blob/4ff9bfd113609fccc51fe314b79447f55a4d6cc6/x-pack/plugins/security_solution/server/endpoint/routes/metadata/index.ts#L18-L19

Does not have to be in this PR. Maybe open a quick ticket to track it.